polling_stations/apps/data_importers/management/commands/import_adur.py

```diff
@@ -2,21 +2,9 @@
 
 
 class Command(BaseXpressDemocracyClubCsvImporter):
-    council_id = "E07000223"
-    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019 (1).tsv"
-    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019 (1).tsv"
-    elections = ["parl.2019-12-12"]
+    council_id = "ADU"
+    addresses_name = "2021-02-15T11:20:20.221841/ADCDemocracy_Club__06May2021.tsv"
+    stations_name = "2021-02-15T11:20:20.221841/ADCDemocracy_Club__06May2021.tsv"
+    elections = ["2021-05-06"]
     csv_delimiter = "\t"
     allow_station_point_from_postcode = False
```

Comment:
This line can be deleted, as that is now the default behaviour - in the past it wasn't so we needed it.